pontoon/api/urls.py

```diff
@@ -14,6 +14,55 @@
         r"^graphql/?$",
         csrf_exempt(GraphQLView.as_view(schema=schema, graphiql=True)),
     ),
+    path(
+        "api/v2/",
+        include(
+            [
+                path(
+                    # Locales
+                    "locales/",
+                    views.LocaleListView.as_view(),
+                    name="locale-list",
+                ),
+                path(
+                    # Locale
+                    "locales/<str:code>",
+                    views.LocaleIndividualView.as_view(),
+                    name="locale-individual",
+                ),
+                path(
+                    # Projects
+                    "projects/",
+                    views.ProjectListView.as_view(),
+                    name="project-list",
+                ),
+                path(
+                    # Project
+                    "projects/<str:slug>",
+                    views.ProjectIndividualView.as_view(),
+                    name="project-individual",
+                ),
+                path(
+                    # Localization
+                    "<str:code>/<str:slug>",
+                    views.ProjectLocaleIndividualView.as_view(),
+                    name="localization-individual",
```

Comment:
Introducing the term "localization" with this specific meaning is a bit superfluous, unless it's intentional?

It's perhaps a bit clumsy, but "project-locale" is how I think this is mostly referred to in the codebase.

Reply:
We use the term "localization" in the [current API](https://github.com/mozilla/pontoon/blob/main/pontoon/api/schema.py) and there's also [pontoon.localizations](https://github.com/mozilla/pontoon/tree/main/pontoon/localizations).

But we should pick either Localization or ProjectLocale and use it consistently in the comment, view name and the URL pattern name.